README.md

```diff
@@ -1,4 +1,15 @@
+### SPANISH PLACEMENT EXAM SCORE
+
+The project is about getting the spanish placement exam scores that is taken by students by enrolling to a specific canvas
+course and sending the to Mpathways. This project get grades from canvas using the CanvasAPI and send them to the Mpathway. API directory
+is used in both cases for getting and sending the grades.
+
+### Project Setup
 1. create venv as `python3 -m venv <directory-to-spe-venv>` and activate the venv
-2. make the api-util-package as part of venv, git clone the project and do `pip install .`. t
-   this should get the api-utils in the site-pacakges 
-3. `pip install -r requirements.txt`
\ No newline at end of file
+2. In future will make use of the `api-util-package` install it by `git clone https://github.com/tl-its-umich-edu/api-utils-python`
```

Comment:
In the (short term) as soon as I actually get through all the PR's I'll put it up on pypi so you can just add it via a requirements.txt like the other modules.